scripts/partition_manager/partition_manager.rst

```diff
@@ -96,33 +99,82 @@ placement: dict
+     The value ``end`` refers to its end address.
      It is not possible to place the partition after ``end`` or before ``start``.
 
       align: dict
          Ensure alignment of start or end of partition by specifying a dict with a ``start`` or ``end`` key respectively, where the value is the number of bytes to align to.
          If necessary, empty partitions are inserted in front of or behind the partition to ensure that the alignment is correct.
          Only one key can be specified.
-         Partitions which directly or indirectly (through spans) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+         Partitions which directly or indirectly (through :ref:`spans <partition_manager_spans>`) share size with the ``app`` partitions can only be aligned if they are placed directly after the ``app`` partition.
+
+.. _partition_manager_spans:
+
+span: list OR span: dict OR span: string
+   This property is used to define container partitions.
+   Its value may be a list, dict, or string.
+
+   Since this property is used to define container partitions, it cannot be used together with the ``placement`` property.
+
+   If the value is a list, its elements are the names of the partitions that should be placed in the container:
+
+   .. code-block:: yaml
+
+      # This partition spans, or contains, partition_1 through partition_n,
+      # in any order:
+      container_partition_name:
+        span: [partition_1, partition_2, ..., partition_n]
+
+   The list elements are interpreted as the set of potential partitions in the container, which the Partition Manager may place in flash in any order. For example, ``partition_2`` could be placed before ``partition_1``.
 
+   Non-existent partitions are removed from the ``span`` list before processing, and partitions with empty ``span`` lists are removed altogether (unless filled by the :ref:`inside property <partition_manager_inside>`).
 
 span: list OR dict: list
    If the value type is a list, this property lists which partitions this partition should span across.
 
    A string formatted value is interpreted as a single item list.
    Partitions with this property are container partitions.
-   Therefore, this property cannot be used together with the ``placement`` property.
 
-   Non-existing partitions are removed from the ``span`` list before processing, and partitions with empty ``span`` lists are removed altogether (unless filled via ``inside``).
+   If the Partition Manager is forced to place a partition that is not declared in the ``span`` list between two partitions that are in the list, the configuration is unsatisfiable and therefore invalid.
+   See :ref:`Span property example 1 <partition_manager_span_ex1>` for an example of an invalid configuration.
+
+   If the value is a dict with key ``one_of``, the semantics are equivalent to a list that contains the first existing partition in the ``one_of``-list:
```

Comment:
This part has been updated in PR https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1770
This seems to have been lost?